examples/sam/distributedtracing/node/src/handlers/sqs-sns-bridge.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
Was the `@newrelic/aws-sdk` plugin  any help here?

https://github.com/newrelic/node-newrelic-aws-sdk/blob/main/lib/sqs.js

Reply:
The agent definitely captures telemetry for the SQS and SNS interactions automatically, which is cool. But it doesn't help with DT.

Reply:
Hmm. That's disappointing but not surprising. This code may serve as a path towards enhancing `@newrelic/aws-sdk` with distributed trace propagation. The Python agent should similarly be propagating `boto3` traces automatically, IMO. This is a lot of code we're expecting the user to write just to get a distributed trace through.

LGTM.